monai/transforms/croppad/dictionary.py

```diff
@@ -54,6 +54,34 @@ def __call__(self, data):
         return d
 
 
+class DivisiblePadd(MapTransform):
+    """
+    Pad the input data, so that the spatial sizes are divisible by `k`.
+
+    Dictionary-based wrapper of :py:class: `monai.transforms.DivisiblePad`.
+    """
+
+    def __init__(self, keys: KeysCollection, k, mode: str = "constant"):
```

Comment:
Should we use `Union[Iterable[str], str]` as you can support both 1 `str` or a `list of str`?
Or put in the doc-string(str or sequence of str)?
Thanks.